terms.html

```diff
@@ -1,254 +1,225 @@
+  <dt><dfn>decentralized identity management</dfn></dt>
 
   <dd>
 <a href="https://en.wikipedia.org/wiki/Identity_management">Identity
-management</a> based on <a>decentralized identifiers</a>.
-Decentralized identity management extends the identifier creation authority
-beyond the traditional roots of trust
-required by <a href="https://en.wikipedia.org/wiki/X.500">X.500
-directory services</a>, the <a href=
-  "https://en.wikipedia.org/wiki/Domain_Name_System">Domain Name
-System</a>, and most national ID systems.
+management</a> based on <a>decentralized identifiers</a>. Decentralized identity
+management extends the identifier creation authority beyond the traditional
+roots of trust required by
+<a href="https://en.wikipedia.org/wiki/X.500">X.500 directory services</a>,
+the <a href="https://en.wikipedia.org/wiki/Domain_Name_System">Domain Name System</a>,
+and most national identification systems.
   </dd>
 
-
-  <dt><dfn data-lt="">decentralized public key infrastructure</dfn> (DPKI)</dt>
+  <dt><dfn>decentralized public key infrastructure</dfn> (DPKI)</dt>
 
   <dd>
-Public key infrastructure based on <a>decentralized identifiers</a>
-and identity records (e.g., <a>DID documents</a>) containing verifiable
+Public key infrastructure based on <a>decentralized identifiers</a> and identity
+records (for example, <a>DID documents</a>) containing verifiable
 <a>public key descriptions</a>.
   </dd>
 
-
-  <dt><dfn data-lt="">delegate</dfn></dt>
+  <dt><dfn data-lt="delegates">delegate</dfn></dt>
 
   <dd>
-An entity who creates a <a>DID</a> and associated <a>DID document</a>
-for a <a>dependent</a> who does not yet have the capacity to control
-the private keys. The dependent is expected to rely on the delegate to safeguard
-the private keys until the dependent can assume control as the <a>DID
-subject</a>.
+An entity who creates a <a>DID</a> and associated <a>DID document</a> for a
+<a>dependent</a> who does not yet have the capacity to control the private keys.
+The <a>dependent</a> is expected to rely on the delegate to safeguard the
+private keys until the <a>dependent</a> can assume control as the
+<a>DID subject</a>.
   </dd>
 
-
-  <dt><dfn data-lt="">dependent</dfn></dt>
+  <dt><dfn data-lt="dependents">dependent</dfn></dt>
 
   <dd>
-A person, organization, or thing whose <a>DID</a> is
-registered and maintained by a delegate because the dependent is not in
-a position to control the private keys. A dependent becomes an
-<a>identity owner</a> when the dependent takes control of the private keys.
+A person, organization, or thing whose <a>DID</a> is registered and maintained
+by a <a>delegate</a> because the dependent is not in a position to control the
+private keys. A dependent becomes an <a>identity owner</a> when the dependent
+takes control of the private keys.
   </dd>
 
-
-  <dt><dfn data-lt="did controller(s)">DID controller</dfn></dt>
+  <dt><dfn data-lt="did controllers">DID controller</dfn></dt>
```

Comment:
```suggestion
  <dt><dfn data-lt="did controller(s)">DID controller</dfn></dt>
```

Reply:
Included "did controller(s)" in the data-lt attribute.